extensions/alacritty/src/utils/theme-switcher.ts

```diff
@@ -0,0 +1,84 @@
+import fs from "fs";
+import path from "path";
+import toml from "@ltd/j-toml";
+import fetch from "node-fetch";
+import { expandPath } from "../utils/path";
+import { showFailureToast } from "@raycast/utils";
+import { getAlacrittyPreferences } from "./get-alacritty-preferences";
+import { AlacrittyConfig, LocalTheme, ExternalTheme } from "../types/theme-switcher";
+
+/**
+ * Get directory to download themes to, from the path of config file.
+ */
+const getThemeDirectory = (configFile: string) => expandPath(`${path.dirname(configFile)}/themes`);
+
+/**
+ * Get display name from file basename.
+ *
+ * @example
+ * getDisplayName('solarized_dark.toml'); // Solarized dark
+ */
+export const getDisplayName = (basename: string) => {
+  const filename = basename.replace(".toml", "");
+
+  return filename
+    .replaceAll(/[-_]/g, " ")
+    .split(" ")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(" ");
+};
+
+/**
+ * Read the current config file.
+ */
+export const readCurrentConfig = (): AlacrittyConfig | null => {
+  try {
+    return toml.parse(
+      fs.readFileSync(expandPath(getAlacrittyPreferences().configFilePath), { encoding: "utf-8" })
+    ) as AlacrittyConfig;
+  } catch (e) {
+    showFailureToast("Failed to read current config file.");
```

Comment:
I think this would be friendlier if it included the currently configured path - on a fresh install it's a little unclear what's going on.